www/about/faq.spt

```diff
@@ -55,6 +55,10 @@ title = _("Frequently Asked Questions")
             '</a>'|safe) }}</p>
     </dd>
 
+    {{ dt(_("Can Liberapay be used to laundry money?"), 'laundry-money') }}
```

Comment:
This is a first draft. Any suggestion is appreciated, but what do you think in making it clear in the FAQ?